### DISCUSSION TITLE
Tabes in pdf

### DISCUSSION THREAD

**Discussion by FilippiChristian** [open-webui/open-webui#4239] [General]
Hello :)
Is there a correct way to submit documents (i used generated pdf) with tables? Seems that tables are not processed.
(Already read, here, questions about database bindings; waiting for it :P )
Thanks 

(Running llama3 on win10 and connecting by openwebui linux)

-->I can not edit the post title. If someone can correct my mistake "tables", thanks.

**Comment by FilippiChristian** [comment]
Seems that I miss something. Also tables in csv format are not processed. Or am I not able to ask model in the right way ?

**Reply by justinh-rahb** [reply]
When PDFs, or CSVs are converted to embeddings in a vector database, they do not retain any concept of formatting. It's strictly text.

**Reply by FilippiChristian** [reply]
Thanks, so it is not possible to achieve (now).
